### PR TITLE
Remove references to turf inside spatial extension

### PR DESCRIPTION
**Pull request by vdelacruzb**

### CODE REVIEW

**Comment by vdelacruzb** [comment]
Adding you @borja-munoz as @martejpad and @Jesus89  are OOO and we need this before tomorrow.

File: app/content/spatial-extension-bq/sql-reference/transformations.md [hunk 1]
```diff
@@ -115,11 +115,11 @@ transformations.ST_CONCAVEHULL(geog, maxEdge, units)
 
 **Description**
 
-Takes a set of points and returns a concave hull Polygon or MultiPolygon. Internally, this uses turf-tin to generate geometries. https://turfjs.org/docs/#concave
+Takes a set of points and returns a concave hull Polygon or MultiPolygon. Internally, this uses turf-tin to generate geometries.
```
Comment (alex-graciano) [inline app/content/spatial-extension-bq/sql-reference/transformations.md:118]: Turf-tin reference

**Review by alex-graciano — CHANGES_REQUESTED** [review]
Fix turf-tin reference

File: app/content/spatial-extension-bq/sql-reference/transformations.md (hunk 1 above)
Comment (vdelacruzb) [reply app/content/spatial-extension-bq/sql-reference/transformations.md:118]: Yes, I was wondering what to do with that one. To me it looks ok, as we are just explaining how it works internally, and we are avoiding links

**Review by borja-munoz — APPROVED** [review]
Looks good to me. The only changed needed is to remove the reference to turf-tin already mentioned by @alex-graciano.

File: app/content/spatial-extension-bq/sql-reference/transformations.md (hunk 1 above)
Comment (borja-munoz) [reply app/content/spatial-extension-bq/sql-reference/transformations.md:118]: I'd remove it. For me it's an implementation detail that we don't need to expose.

**Review by alex-graciano — APPROVED** [review]
LGTM

**Pull request merged**